subsystems/lighting.py

```diff
@@ -0,0 +1,43 @@
+from enum import Enum
+from wpilib._wpilib import AddressableLED
+from constants import lighting_constants
+from hardware import port_constants
+from subsystems.controllers.one_color_controller import OneColorController
+from utils import color
+
+class State(Enum):
+    IDLE = 0
+    SHOOTING = 1
+
+global led_buffer
+global controller
+global __wanted_state
+global is_new_state
+global is_controller_finished
+
+def start(self):
```

Comment:
The `self` parameter is not needed if the function is not in a class. `self` only makes sense for methods inside of classes.

Reply:
oh ok lemme change everything you have send so far